src/devices/Bmi088/Bmi088.cs

```diff
@@ -0,0 +1,301 @@
+﻿using System;
+using System.Buffers.Binary;
+using System.Device.I2c;
+using System.Numerics;
+
+namespace Iot.Device.Bmi088Device
+{
+    /// <summary>
+    /// Based on the C++ implementation on https://github.com/Seeed-Studio/Grove_6Axis_Accelerometer_And_Gyroscope_BMI088
+    /// Product page: https://www.bosch-sensortec.com/products/motion-sensors/imus/bmi088.html
+    /// Data sheet: https://www.bosch-sensortec.com/media/boschsensortec/downloads/datasheets/bst-bmi088-ds001.pdf
+    /// </summary>
+    public class Bmi088 : IDisposable
+    {
```

Comment:
The sensor support as well a temperature sensor. Would be nice as well to get it. It's in the register of the Accelerometer 0x22 and 0x23. 
And if it's implemented, please use UnitsNet to return the temperature, see here for the pattern: https://github.com/dotnet/iot/tree/master/src/devices#using-units